controllers/searchController.js

```diff
@@ -1,15 +1,28 @@
 const searchDao = require('../dao/searchDao');
 const routes = require('express').Router();
 
+const winston = require('winston');
+const logger = winston.createLogger({
+    level: 'info',
+    format: winston.format.json(),
+    defaultMeta: { service: 'searchController' },
+    transports: [
+    //   new winston.transports.File({ filename: 'error.log', level: 'error' }),
+      new winston.transports.File({ filename: 'combined.log' })
+    ]
+});
+
 routes.get('/', function(req, res) {
 
     searchDao.getAll(req.query, function(error, result) {
         if (error) {
+            logger.error(error);
             console.log(error);
```

Comment:
Now that you have a logger, it's probably better to remove all `console.log` calls (or, to retain logging there in the development environment, conditionally add a console transport to Winston, as suggested in its README).

Reply:
Yes, I did. As far as I know